portal_pages/templates/portal_pages/blog_page_add.html

```diff
@@ -0,0 +1,145 @@
+                <input type="text" id="tags_additional" name="tags_additional" />
+            </div>
+
+        </div>
+
+        <div id="required_fields" class="cushion" style="display:none;">
+            <span class="errorlabel">
+                Please fill in all required* fields and enter valid information.
+            </span>
+        </div>
+
+        <div class="cushion">
+            <input type="submit" value="Add My Blog Page" />
+        </div>
+
+    </div></div>
+
+    </form>
+</div></div>
+
+{% endblock %}
+
+{% block extra-js %}
+    <script>
+        $(".datepicker").datepicker({ minDate: 0 });
+        $('#tags').chosen({});
+        tinymce.init({
+            menubar : false,
+            plugins: ["link"],
+            toolbar: [
+            "undo redo | styleselect | bold italic bullist numlist | link" 
+            ],
+            content_css : "/static/css/main.css",
+            style_formats: [
+                {title: 'P', block: 'p'},
+                {title: 'H2', block: 'h2'},
+                {title: 'H3', block: 'h3'},
+                {title: 'H4', block: 'h4'},
+                {title: 'H5', block: 'h5'},
+                {title: 'H6', block: 'h6'},
+            ],
+            selector:'textarea'});
+        $(".addheader").click(function(e){ // toggle collapse/uncollapse
+            var divId = $(e.target).attr("class").split(" ")[1]; // ie. add_focus_area
+            if ($("#"+divId).hasClass("uncollapsed")) {
+                $("#"+divId).hide();
+                $("#"+divId).removeClass("uncollapsed");
+                $("#"+divId).addClass("collapsed");
+            }
+            else {
+                $("#"+divId).show();
+                $("#"+divId).removeClass("collapsed");
+                $("#"+divId).addClass("uncollapsed");
+            }
+        });
+        function checkFormAndSubmit() {
+            formPass = 1;
+            //Clear out the error labels
+            $("label[for='id_title']").removeClass("errorlabel");
+            $("label[for='id_body']").removeClass("errorlabel");
+            $("label[for='id_date']").removeClass("errorlabel");
+            $("#required_fields").hide();
+            //Check for required fields
+            if ($("#id_title").val() == "") {
+                $("label[for='id_title']").addClass("errorlabel");
+                formPass = 0;
+            }
+            if (tinymce.get("id_body").getContent() == "") {
+                $("label[for='id_body']").addClass("errorlabel");
+                formPass = 0;
+            }
+            if ($("#id_date").val() == "") {
+                $("label[for='id_date']").addClass("errorlabel");
+                formPass = 0;
+            }
+            if (formPass) {
+                return true;
+            }
+            else {
+                $("#required_fields").show();
+                return false;
+            }
+        }
+    </script>
```

Comment:
Long term it would be nice to not duplicate essentially the same javascript in 3 places...doesn't need to block merging this but if there is time when all three are in place it might be worth seeing if they can be consolidated.